core/src/Cuda/Kokkos_Cuda_KernelLaunch.hpp

```diff
@@ -19,9 +19,14 @@
 #include <Cuda/Kokkos_Cuda_GraphNodeKernel.hpp>
 #include <Cuda/Kokkos_Cuda_BlockSize_Deduction.hpp>
 
+#if defined(KOKKOS_COMPILER_NVCC) && !defined(KOKKOS_ARCH_MAXWELL) && \
+    !defined(KOKKOS_ARCH_PASCAL)
+#define KOKKOS_IMPL_USE_GRID_CONSTANT
+#endif
```

Comment:
Any reason not to use `KOKKOS_IMPL_USE_GRID_CONSTANT` in `core/src/Cuda/Kokkos_Cuda_Instance.hpp` ? I mean, defining it in this file or in `core/src/Cuda/Kokkos_Cuda_Instance.hpp` does not change the visibility of this macro, but improves the consistency.